src/keycloak/chart/values.schema.json

```
@@ -140,6 +140,14 @@
     "autoscaling": {
       "$ref": "#/definitions/autoscalingSchema"
     },
+    "highAvailability": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "enabled": { "type": "boolean" },
+        "maxReplicas": { "type": "number" }
```

```suggestion
        "maxReplicas": { 
           "type": "number",
           "not": {
            "multipleOf": 2
           }
        }
```

I think you can also add this little check to make sure people aren't deploying even number of instances

---

Great suggestion! Added!

---

I implemented this in a slightly different place as this part of the schema also applies to the Waypoint (where we don't have these restrictions).